tier2/{{cookiecutter.project_slug}}/.github/workflows/contributors.yml

```diff
@@ -0,0 +1,55 @@
+on:
+    push:
+        branches:
+            - main
+    workflow_dispatch:
+
+jobs:
+    update-contributors:
+        runs-on: ubuntu-latest
+        name: Update contributors info in MAINTAINERS.md
+        permissions:
+            contents: write
+            pull-requests: write
+        steps:
+            # Update contributor list
+            - name: Contribute List
+              uses: akhilmhdh/contributors-readme-action@v2.3.10
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              with:
+                  # https://github.com/marketplace/actions/contribute-list#optional-parameters
+                  readme_path: MAINTAINERS.md
+                  use_username: false
+                  commit_message: "BOT: Update contributors info in MAINTAINERS.md"
+
+            # Update contributor count
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Pull changes from contributors-readme-action
+              run: |
+                  git pull
+
+            - name: Get repository contributors count
+              id: get_contributors
+              run: |
+                  REPO_OWNER=$(echo $GITHUB_REPOSITORY | cut -d'/' -f1)
+                  REPO_NAME=$(echo $GITHUB_REPOSITORY | cut -d'/' -f2)
+                  CONTRIBUTORS=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" "https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/contributors?per_page=100" | jq '. | length')
```

Comment:
This request will only get the amount of contributors up to 100.

A better way to get the count would be to use a graphql query to the [GitHub graphql api](https://docs.github.com/en/graphql). Here is the graphql query that we use to get counts in the metrics website stack as an example:

```gql

query ($repo: String!, $owner: String!) {
  repository(name: $repo, owner: $owner) {
    description,
    forkCount,
    forkingAllowed,
    stargazerCount,
    
    pullRequests(first: 1)
    {
      totalCount
    },
    mergedPullRequests: pullRequests(first: 1, states: MERGED)
    {
      totalCount
    },
    closedPullRequests: pullRequests(first: 1, states: CLOSED)
    {
      totalCount
    },
    openPullRequests: pullRequests(first: 1, states: OPEN)
    {
      totalCount
    },
    issues(first: 1)
    {
      totalCount
    },
    openIssues: issues(first: 1, states: OPEN)
    {
      totalCount
    },
    closedIssues: issues(first: 1, states: CLOSED)
    {
      totalCount
    },
    watchers(first: 1)
    {
      totalCount
    }
    defaultBranchRef
    {
      name,
      target
      {
        ... on Commit
        {
          history(first: 1)
          {
            totalCount
          }
          
        }
      }
    }
  }
}

```

Reply:
Fixed!